01week/rockPaperScissors.js

```diff
@@ -43,7 +71,20 @@ if (typeof describe === 'function') {
       assert.equal(rockPaperScissors('Paper', 'SCISSORS'), "Hand two wins!");
       assert.equal(rockPaperScissors('rock ', 'sCiSsOrs'), "Hand one wins!");
     });
-  });
+    it('should detect all possible win options for hand 1', () => {
+      assert.equal(rockPaperScissors('rock', 'scissors'), "Hand one wins!");
+      assert.equal(rockPaperScissors('paper', 'rock'), "Hand one wins!");
+      assert.equal(rockPaperScissors('scissors', 'paper'), "Hand one wins!");
+    });
+    it('should detect all possible win options for hand 2', () => {
+      assert.equal(rockPaperScissors('scissors', 'rock'), "Hand two wins!");
+      assert.equal(rockPaperScissors('rock', 'paper'), "Hand two wins!");
+      assert.equal(rockPaperScissors('paper', 'scissors'), "Hand two wins!");
+    });
+    it('should detect a valid entry - rock, paper, scissors', () => {
+      assert.equal(rockPaperScissors('unicorn', 'flowers'), "This is an invalid entry");
```

Comment:
@AFutureWebDev 
Thanks for adding this, a lot of people forgot about this tests.